services/SignupMetricsService.ts

```diff
@@ -49,6 +89,7 @@ export default class SignupMetricsService {
 
     signups.forEach((signup: Signup) => {
       const existingSignup = signupsByEmail[signup.email.toString()];
```

Comment:
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/DynamoDB/Converter.html `output` might eliminate the need for eslint disable

Reply:
Eventually we'll want to take a look at this more closely because it really does look like a completely unnecessary check since `signupsByEmail` is always instantiated as `{}` up on line 85.